definitions/features/hammer.rb

```diff
@@ -197,9 +200,22 @@ def load_from_file(file_path)
     end
   end
 
+  def username_from_answers
+    return nil unless feature(:installer)
```

Comment:
We may want to move the logic to the installer feature and have here just
```ruby
return nil unless feature(:installer)
feature(:installer).initial_admin_username
```

Also, would it work if we skip the expensive check_min_version and use just
`feature(:installer).answers['foreman']['initial_admin_username'] || feature(:installer).answers['foreman']['admin_username']`?